### PR TITLE
Add definitions for blob-stream 

### PR DESCRIPTION
**Pull request by erichillah**
https://www.npmjs.com/package/blob-stream

### CODE REVIEW

File: blob-stream/blob-stream.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+
+declare function BlobStream(): BlobStream.IBlobStream;
+
+declare module BlobStream {
+
+    interface IBlobStream extends NodeJS.WritableStream{
+        toBlob(type: string): Blob;
```
Comment (horiuchi) [inline blob-stream/blob-stream.d.ts]: I saw the example of `blob-stream`.
Maybe, this argument should be optional?

**Comment by erichillah** [comment]
Definetely, just pushed the changes.
Thanks.

**Comment by vvakame** [comment]
*blob-stream/blob-stream.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/blob-stream
  * http://bower.io/search/?q=blob-stream
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (blob-stream/blob-stream-tests.ts or others)
* [ ] pass the Travic-CI test?

**Comment by vvakame** [comment]
@erichillah thanks mate!

**Pull request merged**